Source/SPStorkController/TransitioningDelegate/SPStorkPresentationController.swift

```diff
@@ -108,17 +108,35 @@ class SPStorkPresentationController: UIPresentationController, UIGestureRecogniz
             let tap = UITapGestureRecognizer.init(target: self, action: #selector(self.dismissAction))
             tap.cancelsTouchesInView = false
             self.indicatorView.addGestureRecognizer(tap)
+            self.indicatorView.accessibilityLabel = NSLocalizedString("Close", comment: "")
             presentedView.addSubview(self.indicatorView)
             self.indicatorView.translatesAutoresizingMaskIntoConstraints = false
             self.indicatorView.widthAnchor.constraint(equalToConstant: 36).isActive = true
             self.indicatorView.heightAnchor.constraint(equalToConstant: 13).isActive = true
             self.indicatorView.centerXAnchor.constraint(equalTo: presentedView.centerXAnchor).isActive = true
             self.indicatorView.topAnchor.constraint(equalTo: presentedView.topAnchor, constant: 12).isActive = true
+
+            if UIAccessibility.isVoiceOverRunning {
```

Comment:
I only added this conditional because I wasn't sure if anyone relies on the rest of the top of the view being open for taps.